discovery-provider/src/api_helpers.py

```diff
@@ -23,5 +60,44 @@ def success_response(response_entity=None, status=200):
 
     response_dictionary['latest_indexed_block'] = (int(latest_indexed_block) if latest_indexed_block else None)
     response_dictionary['latest_chain_block'] = (int(latest_chain_block) if latest_chain_block else None)
+    response_dictionary['version'] = disc_prov_version
+    response_dictionary['owner_wallet'] = shared_config['delegate']['owner_wallet']
 
-    return jsonify(response_dictionary), status
+    return response_dictionary
+
+# Generate signature and timestamp using data and adding those fields to response dict
+def generate_and_set_sig_and_timestamp(response_dict, data):
```

Comment:
i'd advise against mutator functions like this where you take a property and change it without returning. i would advocate for pure functions where there's no mutations of inputs and the output is deterministic. this function may not even be necessary honestly